src/components/album/album-comments/comment-new/comment-new.component.js

```diff
@@ -0,0 +1,27 @@
+import template from './comment-new.html';
+
+export const CommentNewComponent = {
+	template,
+	bindings: {
+		addComment: '&'
+	},
+	controller: class CommentNewComponent{
+		constructor(){
+			this.email = '';
+			this.text = '';
+		}
+
+		newComment(){
+			if(this.email && this.text.length){
+				let newComment = {
+				  albumId: '',
+				  email: this.email,
+				  text: this.text,
+				  avatar: '',
```

Comment:
It's like the blueprint of the object, but yes, I will set the property when I have already the value